scripts/upgrade

```diff
@@ -75,6 +77,14 @@ if [ -e "$bdd" ]; then
   cp -a $bdd /tmp/association.sqlite
 fi
 
+if  [ -d "$data" ]; then
+  cp -ar $data /tmp/data
+fi
+
+if [ -d "$skel_dist" ]; then
+  cp -ar $skel_dist /tmp/skel-dist
```

Comment:
> Nitpicking: the `-r` option is included in the `-a` option
cp
-a | Copies the file with the same permission settings and metadata as the original.
-r | Directories are copied recursively including subdirectories.

Are you sure I don't need add  `-r` ?

> I think it's a good practice when dealing with paths and variable names is generally to surround with quotes.

I just know that like this it works, do you think it would be better to say `cp -a "$skel_dist" /tmp/skel-dist` ?

Reply:
> Are you sure I don't need add *-r* ?

The documentation states :
       -a, --archive
              same as -dR --preserve=all

And later:
       -R, -r, --recursive
              copy directories recursively

Source: https://www.man7.org/linux/man-pages/man1/cp.1.html

---

> do you think it would be better to say *cp** -a "$skel_**dist**" /**tmp**/**skel-dist* ?

Yes I do :)

Reply:
Ok, I push this https://github.com/YunoHost-Apps/garradin_ynh/pull/70/commits/58e2a40af035ffec33da102ef38625d2b18af0e9

Reply:
resloved with #73